src/airflow_tools/providers/filesystem/operators/filesystem.py

```diff
@@ -57,52 +120,43 @@ def execute(self, context):
             destination_fs_hook.write(data, full_destination_path)
 
 
-class SQLToFilesystem(BaseOperator):
-    """
-    Copies data from a SQL query to a filesystem.
-    """
+class FilesystemDeleteOperator(BaseOperator):
+    template_fields = ('filesystem_path',)
 
-    template_fields = (
-        'sql',
-        'destination_path',
-    )
+    def __init__(self, filesystem_conn_id: str, filesystem_path: str, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.filesystem_conn_id = filesystem_conn_id
+        self.filesystem_path = filesystem_path
+
+    def execute(self, context: 'Context'):
+        filesystem_protocol = FilesystemFactory.get_data_lake_filesystem(
+            connection=BaseHook.get_connection(self.filesystem_conn_id),
+        )
+        if filesystem_protocol.check_prefix(self.filesystem_path):
+            filesystem_protocol.delete_prefix(self.filesystem_path)
```

Comment:
I think that the `delete_prefix` shouldn't fail if the path does not exist. If a specific filesystem fails in that scenario I think we should delegate the checking to that filesystem's `delete_prefix` method, since some filesystems don't require a check.

Reply:
Yes so I think this check should be done inside the volume's filesystem implementation, in the filesystem API we expose I think deleting a prefix shouldn't fail, o al menos tener el parámetro `exist_ok` como Pathlib.